### DISCUSSION TITLE
24GHz mmWave Human Static Presence | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1111] [Q&A]
# 24GHz mmWave Human Static Presence | Seeed Studio Wiki

24GHz mmWave Human Static Presence

https://wiki.seeedstudio.com/Radar_MR24HPB1/

**Comment by Zaimalik** [comment]
Can this sensor detect behind wall? and for how many meters? 
Regards.

**Reply by Divyanshuvyas007** [reply]
Hey I wanna know this too did you get any solution???

**Comment by viralgala7** [comment]
We are using MR24HPB1 sensor for static presence detection & it was working fine, suddenly one day it started giving pattern as, 55 AA 03 07 0005 65 01 00 01 76 55 AA 03 07 00 05 83 04 00 01 00 96 55 AA 03 01 00 2A 7B 22 70 22 08 02 02 00 04 00 00 00 03 1C......and so on which are not having the RX data format as mentioned in datasheet, Length bytes gives AA 03 & data terminates 11 bytes pack, please check & help me decode this, is there anything gone wrong with the chip, can it be reprogrammed?
